lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml

```diff
@@ -373,7 +373,13 @@ definitions:
     required:
       - next_page_token
       - subscribers
+      - total_count
```

Comment:
I'm fine with the way we're doing this count (making it a separate call to configurator) for the sake of simplicity of implementation. That said, if we do keep it as a separate configurator call, this field needs to be `estimated_total_count`, to indicate that it's not a reliable count of exactly how many subscribers there are

Reply:
I don't think we need to be extremely precise in naming here. Calling this estimated_total_count maybe more confusing rather than clarifying the implementation detail involved here. Since it is more an informational field in UI, I would prefer ignoring this detail and just calling this total_count.